### PR TITLE
Allow passing flatListProps

### PR DESCRIPTION
**Pull request by vbuch**
Set the windowSize property of FlatList to default 3.
#19

### CODE REVIEW

File: src/Gallery.js [hunk 1]
```diff
@@ -19,13 +19,17 @@ export default class Gallery extends PureComponent {
...
+        flatListProps: PropTypes.object,
     };
 
     static defaultProps = {
         removeClippedSubviews: true,
         imageComponent: undefined,
-        scrollViewStyle: {}
+        scrollViewStyle: {},
+        flatListProps: {
+            windowSize: 3,
```
Comment (Exilz) [inline src/Gallery.js]: Maybe we should set this `windowSize` value directly where we render `Flatlist` to make sure this isn't overriden when the users use `flatListProps` prop.
Or, we would have to merge the default value and the user's one.

Comment (vbuch) [reply src/Gallery.js]: Yup. Makes sense. The soonest I'll be able to fix this is on Monday.

File: src/Gallery.js [hunk 2]
```diff
@@ -265,9 +267,12 @@ export default class Gallery extends PureComponent {
             gestureResponder = {};
         }
 
+        const flatListProps = Object.assign(DEFAULT_FLAT_LIST_PROPS, this.props.flatListProps)
```
Comment (Exilz) [inline src/Gallery.js]: Much better ! Could you use the spread operator here to keep the codebase more consistant ?

**Comment by Exilz** [comment]
Look good, I'll test this on the demo and merge it tomorrow.

**Comment by Exilz** [comment]
Can you add it in the README's props table as well ?

**Comment by vbuch** [comment]
Great! Thanks!

**Comment by vbuch** [comment]
Any chance of pushing to npm, @Exilz ?

**Comment by Exilz** [comment]
@vbuch published as `2.1.0`

**Comment by vbuch** [comment]
Thanks a bunch!

**Pull request merged**